internal/guest/runtime/hcsv2/workload_container.go

```diff
@@ -140,6 +173,8 @@ func setupWorkloadContainerSpec(ctx context.Context, sbid, id string, spec *oci.
 	// Clear the windows section as we dont want to forward to runc
 	spec.Windows = nil
 
+	// Displat mounts
+	displayMounts(ctx, spec)
```

Comment:
Was this just for debugging?

Reply:
yes, will remove it, when I publish the pr.